pkg/util/pki/generate.go

```diff
@@ -44,6 +45,9 @@ const (
 	ECCurve384 = 384
 	// ECCurve521 represents a secp521r1 / NIST P-521 ECDSA key.
 	ECCurve521 = 521
+
+	// Ed25519
+	Ed25519 = 25519
```

Comment:
suggestion: I don't think we need to add this; `v1.Ed25519KeyAlgorithm` is enough to uniquely identify which curve we're using and the key size doesn't change